packages/react/src/toggle-button-group/item/ToggleButtonGroupItem.tsx

```diff
@@ -0,0 +1,145 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { useComponentRenderer } from '../../utils/useComponentRenderer';
+import type { BaseUIComponentProps } from '../../utils/types';
+import { CompositeItem } from '../../composite/item/CompositeItem';
+import { useToggleButtonGroupRootContext } from '../root/ToggleButtonGroupRootContext';
+import { useToggleButtonGroupItem } from './useToggleButtonGroupItem';
+
+const customStyleHookMapping = {
+  disabled: () => null,
+};
+/**
+ *
+ * Demos:
+ *
+ * - [ToggleButtonGroup](https://base-ui.com/components/react-toggle-button-group/)
+ *
+ * API:
+ *
+ * - [ToggleButtonGroupItem API](https://base-ui.com/components/react-toggle-button-group/#api-reference-ToggleButtonGroupItem)
+ */
+const ToggleButtonGroupItem = React.forwardRef(function ToggleButtonGroupItem(
```

Comment:
I started off doing that but ran into an issue that made it more practical to have this `Item` subcomponent, let me look up what the actual issue was 

@colmtuite is this an issue for you?

Reply:
The main issue was, if there wasn't the `ToggleButtonGroup.Item` component, the `value` becomes optional for `ToggleButton.Root`, even when it's in a toggle group

With the separate `.Item`, it's easier to ensure all toggle buttons in a group have a `value`

Unlike Accordion/Tabs, it makes less sense to not provide your own value and use the index position fallback value (it would still work 🤷 )

Reply:
We decided to align the API with RadioGroup/CheckboxGroup for consistency, as it's relatively easy for users to enforce a required `value` for toggle buttons inside a group themselves